src/primitives/transaction.cpp

```diff
@@ -1047,6 +1047,14 @@ bool CTransaction::IsVersionStandard(int nHeight) const {
 
 bool CTransaction::ContextualCheck(CValidationState& state, int nHeight, int dosLevel) const
 {
+    // after this fork point we do not have any transaction flowing but coinbase txes
+    if (!IsCoinBase() && ForkManager::getInstance().areTransactionsStopped(nHeight)) {
```

Comment:
Is my understanding here correct that IsCoinBase covers both coinbase transactions and certificates, or only coinbase transactions?

If the intended behavior is to still allow certificates, I'd like to see this covered in the RPC test (certificates still work after the fork).

Reply:
Actually it covers only coinbase txs, certificates are handled in another method. But it definitely makes sense testing that certificates are working after the forks. I will add that to the py test, thanks.

Reply:
Test added